flow/ProtocolVersions.cmake

```diff
@@ -13,6 +13,7 @@ set(FDB_PV_FUTURE_VERSION                       "0x0FDB00B073000000LL")
 set(FDB_PV_MIN_COMPATIBLE_VERSION               "0x0FDB00B071000000LL")
 set(FDB_PV_MIN_INVALID_VERSION                  "0x0FDB00B074000000LL")
 set(FDB_PV_LEFT_MOST_CHECK                      "0x0FDB00B100000000LL")
+set(FDB_PV_LSB_MASK                             "0xF0FFFFLL")
```

Comment:
This single value now replaces what was two different values with the same value, is that intentional?

Reply:
It was an original PR that changed the first one, and I missed this one. So I'm moving both out to the cmake file.